connectivity/check/deployment.go

```diff
@@ -911,9 +911,8 @@ func (ct *ConnectivityTest) deploy(ctx context.Context) error {
 
 				svc := newService(echoExternalNodeDeploymentName,
 					map[string]string{"name": echoExternalNodeDeploymentName, "kind": kindEchoExternalNodeName},
-					map[string]string{"kind": kindEchoExternalNodeName}, "http", 8080)
+					map[string]string{"kind": kindEchoExternalNodeName}, "http", 8080, "ClusterIP")
 				svc.Spec.ClusterIP = corev1.ClusterIPNone
-				svc.Spec.Type = corev1.ServiceTypeClusterIP
```

Comment:
With this PR, we lose the type information from corev1, but I think that's reasonable in this case since these strings are so well-known and would fail to provision if misconfigured.